backend/data/data.json

```diff
@@ -0,0 +1,4671 @@
+[
```

Comment:
I don't think this should be publicly stored on github? There's real stories here, right - this isn't mock data.

Reply:
I deleted the file and purged it from the git history. We can distribute it privately if needed.